### ISSUE TITLE
getting error when downloading SRR files - "fasterq-dump.2.9.1 sys: timeout exhausted while reading file"

### ISSUE THREAD

**Issue by frymor** [ncbi/sra-tools#206]
Hi,

I'm downloading some files from the SRA DB. For some of the files I keep getting  strange error message. I'm not even sure, The files are downloaded, some of them are not downloaded completely, other do, but I can't see any pattern.
The command i use is 
```
fasterq-dump -p -e 10  --split-files -O fastqFiles  SRRXXXXXX
```

The error i keep getting (for some of the files is:

```
join   :|-------- 16.62%2019-06-03T09:57:49 fasterq-dump.2.9.1 sys: timeout exhausted while reading file within network system module - mbedtls_ssl_read returned -76 ( NET - Reading information from the socket failed )                                                   -------------- 44.02%2019-06-03T10:00:11 fasterq-dump.2.9.1 sys: timeout exhausted while reading file within network system module - mbedtls_ssl_read returned -76 ( NET - Reading information from the socket failed )                                                             ---------------------------- 100.00%
concat :|-------------------------------------------------- 100.00%
spots read      : 10,452,521
reads read      : 20,905,042
reads written   : 20,905,042
```
In this example the error was shown, but (probably) the file was completely downloaded.

Any ideas, why this happens or how to fix it?

thanks
Assa

**Comment by kwrodarmer** [comment]
Hi @frymor,

> Any ideas, why this happens or how to fix it?

We wrote a Wiki page about it:
https://github.com/ncbi/sra-tools/wiki/Network-and-Connection-Errors

There's very little we can do with network errors other than to make our tools try again and again, which is what they do. As far as the report you get in the middle when we announce a network error, we have opted in the past to give more output in the chance that some user report would help us to identify ways of better handling the errors. Unfortunately, this also can give the false impression that the overall operation has failed. Beginning with release 2.10.0 we are going to reduce the frequency of these spurious reports.

To be clear, the reports are real; the error does occur. But the underlying VDB system responds by restarting network operations and in most cases recovers without damage. When recovery is not possible, the tool will exit with an error status. When the tool exits and reports success, it indicates that it was able to recover from transient network errors.
